torch/csrc/jit/tracer.h

```diff
@@ -114,6 +118,21 @@ TORCH_API void addInputs(
     const c10::optional<at::ScalarType>& value);
 TORCH_API void addInputs(Node* n, const char* name, at::Generator* value);
 
+template<typename T>
+TORCH_API void addInputs(
+    Node* n,
+    const char* name,
+    const std::vector<T>& value) {
+  AT_ERROR("Tracing generic lists currently not supported!");
```

Comment:
Remember that a "generic list" is a thing that's only internal to our compiler. When you're writing user-facing error messages avoid using internal terms, and tell them something they would understand without a deep code dive.